backend/inspirehep/records/api/literature.py

```diff
@@ -309,7 +309,7 @@ def _update_file_entry(self, file_data, add_file_func):
         try:
             add_file_func(**file_data)
         except ValueError:
```

Comment:
I think it makes sense to keep throwing ValueError from schemas in this case, as we are trying to populate literature with wrong data.
In our case, we want to skip over those files, so we catch it and just log warning

Reply:
you are right!